python/setup.py

```diff
@@ -1,6 +1,6 @@
 import setuptools
 
-with open("../README.md", "r", encoding="utf-8") as fh:
+with open("../README.md", "r") as fh:
```

Comment:
Why was this needed? I think this means it won't work with utf-8 characters.

Reply:
I can undo this, it was just causing problems for me while compiling on my OSX laptop.